src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/ReportDocHelper.java

```diff
@@ -63,6 +68,7 @@ public class ReportDocHelper {
     private static final String CASE_REFERENCE = "{\"Case_Reference\":\"";
     private static final String CANNOT_CREATE_REPORT_DATA_EXCEPTION = "Unable to create report data";
     private static final String LISTING_DATA_STATE_EXCEPTION = "ListingData is not instanceof ";
+    private static final int FIVE = 5;
```

Comment:
The name of this constant could do with being something more meaningful

Reply:
I agree.